8.gtex-interpret/scripts/utils.py

```diff
@@ -47,24 +48,24 @@ def load_weight_matrix(dataset, z_dim, seed, shuffled=False):
     return weight_df
 
 
-def load_enrichment_results(dataset, z_dim, metaedge, algorithm=None,
-                            feature=None, seed=None, shuffled=False):
+def load_enrichment_results(
+    dataset, z_dim, metaedge, algorithm=None, feature=None, seed=None, shuffled=False
```

Comment:
minor,  you could create newlines to make this line shorter

Reply:
:+1: